.github/actions/rancher-interaction/action.yml

```diff
@@ -0,0 +1,52 @@
+name: "rancher-interaction"
```

Comment:
just `rancher`?

maybe in the future we could add more actions related to rancher managed clusters